include/units.h

```diff
@@ -956,7 +904,7 @@ namespace units
 		static_assert(traits::is_ratio_v<PiExponent>, "Template parameter `PiExponent` must be a `std::ratio` representing the exponents of Pi the unit has.");
 		static_assert(traits::is_ratio_v<Translation>, "Template parameter `Translation` must be a `std::ratio` representing an additive translation required by the unit conversion.");
 
-		using dimension_type = units::dimension_t<Exponents...>;
+		using dimension_type = dimension_t<Exponents...>;

```

Comment:
Just above, in the template parameter list, `dimention_t` isn't prefixed with `units::`. Should I have done the change the other way around?

Reply:
nope, it was inconsistent and verbose for no reason.